src/main/java/org/kainos/ea/core/DeliveryEmployeeValidator.java

```diff
@@ -0,0 +1,52 @@
+package org.kainos.ea.core;
+
+import org.kainos.ea.cli.DeliveryEmployee;
+import org.kainos.ea.cli.DeliveryEmployeeRequest;
+import org.kainos.ea.cli.DeliveryEmployeeUpdateRequest;
+import org.kainos.ea.client.FailedToGetDeliveryEmployee;
+import org.kainos.ea.db.DeliveryDao;
+
+import java.sql.SQLException;
+
+public class DeliveryEmployeeValidator {
+    DeliveryDao deliveryDao;
+
+    public DeliveryEmployeeValidator(DeliveryDao deliveryDao) {
+        this.deliveryDao = deliveryDao;
+    }
+
+    public String doesEmployeeExist(int id) throws SQLException {
```

Comment:
This function has a yes or no answer so could this return something other then string and null